shared/src/test/diff/qq/Hygiene.mls

```diff
@@ -0,0 +1,63 @@
+:NewDefs
+:NoJS
+
+
+// Test 1
+let code_test = code"let x0 = 1 in let x1 = 10 in let x2 = 100 in x0 + x1 + x2"
+//│ let code_test: Code[Int, nothing]
+
+run(code_test)
+//│ Int
+
+
+// Test 2: this is hygienic because the child Quoted is a descendant of the parent Quoted in the AST
```

Comment:
I don't think "descendant" is standard terminology. What do you mean here?

Reply:
I think it is an obsolete comment from the old qq PR. Can we remove it (and another one mentioned above), or should we supplement something here?

Reply:
No, I think it is from the #164 so I don't know either. Anyway, it is removed.

Reply:
Oh! That explains the weird, mostly nonsensical wording lol